assets/setup-wizard/ready/index.test.js

```diff
@@ -140,4 +191,35 @@ describe( '<Ready />', () => {
 
 		expect( submitMock ).not.toBeCalled();
 	} );
+
+	it( 'Should run sample installation', async () => {
+		jest.useFakeTimers();
+		window.location.assign = jest.fn();
+
+		const startPromise = Promise.resolve( { id: 1 } );
+		const logsPromisePending = Promise.resolve( {
+			status: { status: 'pending' },
+		} );
+		const logsPromiseCompleted = Promise.resolve( {
+			status: { status: 'completed' },
+		} );
+
+		apiFetch.mockImplementationOnce( () => startPromise );
```

Comment:
I recently found a handy [shortcut](https://jestjs.io/docs/en/mock-function-api#mockfnmockresolvedvaluevalue) for this: `.mockResolvedValueOnce({ id: 1 })`

Reply:
Cool! TIL!
https://github.com/Automattic/sensei/pull/3507/commits/117790d17be4a0967a56d2d612bb937a30c49d70